list.go

```diff
@@ -0,0 +1,85 @@
+// +build linux
+
+package main
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"text/tabwriter"
+
+	"github.com/Sirupsen/logrus"
+	"github.com/codegangsta/cli"
+)
+
+var listCommand = cli.Command{
+	Name:  "list",
+	Usage: "lists containers started by runc with the given root",
+	Action: func(context *cli.Context) {
+
+		// preload the container factory
+		if factory == nil {
+			err := factoryPreload(context)
+			if err != nil {
+				logrus.Fatal(err)
+				return
+			}
+		}
+
+		// get the list of containers
+		root := context.GlobalString("root")
+		absRoot, err := filepath.Abs(root)
+		if err != nil {
+			logrus.Fatal(err)
+			return
+		}
+		list, err := ioutil.ReadDir(absRoot)
+
+		w := tabwriter.NewWriter(os.Stdout, 12, 1, 3, ' ', 0)
+		fmt.Fprint(w, "ID\tSTARTED\tSTATUS\tPID\tROOTFS\n")
```

Comment:
Order:  I have no preference on the order so... sure changed.

Thoughts on BUNDLE vs. ROOTFS: 
We are making container id a required field on all of the runc commands that previously needed you to either be in the bundle or pass the old bundle as an optional parameter.  Decision made at the last face to face.  Thus the utility of the bundle as a thing you have to know from one command to the next on a running container is changing.

I think there is good utility for knowing the rootfs of the currently running container.  
The bundle on the other hand, is an optional path argument to runc start/create.  The bundle path is not being stored ATM (I don't believe).  We take a snap shot of everything that was there at start/create and then we are done with it.  Eg. we make a copy of the config.json and that copy is stored as state.  The original is not used again unless they start a new container from the same bundle. From one runc start to the next, the config.json in the bundle directory could have been edited and it's not required to still be there.  

So... I'd rather have either an option on list (-d etc.) to output more detail in which we add things like BUNDLE and maybe a few other items... or have a different command entirely that dumps as much or as little as you might like to see about a particular container.